packages/std/src/imports.rs

```diff
@@ -26,6 +26,9 @@ const HUMAN_ADDRESS_BUFFER_LENGTH: usize = 90;
 // A complete documentation those functions is available in the VM that provides them:
 // https://github.com/CosmWasm/cosmwasm/blob/v1.0.0-beta/packages/vm/src/instance.rs#L89-L206
 extern "C" {
+    #[cfg(feature = "abort")]
```

Comment:
Simply requiring this import (by setting the abort feature) will cause the contract to fail to upload if the VM doesn't support this.

As it is an agreement between the VM and the contract, this should be sufficient and we can remove the required feature, making it an even smoother integration